src/Sylius/Bundle/PayumBundle/Action/ConvertPaymentToOfflineAction.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+* This file is part of the Sylius package.
+*
+* (c) Paweł Jędrzejewski
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Sylius\Bundle\PayumBundle\Action;
+
+use Payum\Core\Action\ActionInterface;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Core\Model\PaymentInterface;
+use Payum\Core\Request\Convert;
+use Payum\Offline\Constants;
+
+/**
+ * @author Antonio Peric <antonio@locastic.com>
+ */
+class ConvertPaymentToOfflineAction implements ActionInterface
+{
+    /**
+     * {@inheritDoc}
+     *
+     * @param Convert $request
+     */
+    public function execute($request)
+    {
+        RequestNotSupportedException::assertSupports($this, $request);
+
+        $request->setResult([
+            Constants::FIELD_PAID => true,
```

Comment:
@antonioperic I had to change the default behavior, to make tests pass, if you need pending, overwrite this converter in your app code.